clients/client/src/main/java/org/projectnessie/client/ClientTreeApi.java

```diff
@@ -134,6 +134,7 @@ public LogResponse getCommitLog(String ref, CommitLogParams params)
         .queryParam("pageToken", params.getPageToken())
         .queryParam("before", null != params.getBefore() ? params.getBefore().toString() : null)
         .queryParam("after", null != params.getAfter() ? params.getAfter().toString() : null)
+        .queryParam("cel_expr", params.getCelExpr())
```

Comment:
Here and elsewhere I don't like the name `cel_expr`. I would prefer `query_expression` or `expression` or something. I don't think it is relevant that cel is being used. We want to advertise to clients that there is an expression parameter available only.

I think this applies to everythign below (eg cli, python etc)  I will just mention it here to keep the review clean

Reply:
updated to `query_expression`